invokeai/frontend/web/src/features/nodes/components/ui/ClearNodesButton.tsx

```diff
@@ -10,31 +22,68 @@ import IAIIconButton from 'common/components/IAIIconButton';
 const ClearNodesButton = () => {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
```

Comment:
I think it's better to use Chakra's useDisclosure to handle the modal status. You can find documentation on that here. https://chakra-ui.com/docs/components/alert-dialog/usage

Reply:
@blessedcoolant ok changed it to useDisclosure instead of useState